org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/Protocol.xtend

```diff
@@ -735,6 +753,11 @@ class TextDocumentClientCapabilities {
 	 * Capabilities specific to `textDocument/publishDiagnostics`.
 	 */
 	PublishDiagnosticsCapabilities publishDiagnostics
+
+	/**
+	 * Capabilities specific to {@code textDocument/semanticHighlighting}.
+	 */
+	SemanticHighlightingCapabilities semanticHighlightingCapabilities
```

Comment:
In [the proposal](https://github.com/kittaakos/vscode-languageserver-node/blob/e881846b5d93d4732c9307429f402814d382e092/protocol/src/protocol.semanticHighlighting.proposed.md) this is just a Boolean.

Reply:
Thanks! I have aligned the implementation with the most recent state of the proposal.